Apps/ValidationTests/Win32/pch.h

```diff
@@ -0,0 +1,81 @@
+// stdafx.h : include file for standard system include files,
```

Comment:
hmm, i don't think we use pch.h anymore... should remove pch.h and pch.cpp from both projects.